src/main/java/org/cascadebot/cascadebot/commands/moderation/ModlogDisableSubCommand.java

```diff
@@ -0,0 +1,143 @@
+            } else {
+                failedEvents.add(event);
+            }
+        }
+        List<Page> pageList = new ArrayList<>();
+        if (failedEvents.size() == modlogEvents.size()) {
+            EmbedBuilder embedBuilder = new EmbedBuilder();
+            embedBuilder.setColor(MessageType.DANGER.getColor());
+            embedBuilder.setTitle("Failed to disable all specified events");
+            embedBuilder.setDescription("Failed to disable or find all specified events. Refer to the other pages for more info");
+            pageList.add(new PageObjects.EmbedPage(embedBuilder));
+        } else {
+            EmbedBuilder embedBuilder = new EmbedBuilder();
+            embedBuilder.setColor(MessageType.SUCCESS.getColor());
+            embedBuilder.setTitle("Successfully disabled events!");
+            embedBuilder.setDescription("Some events may have still failed to disable, check other pages for more info");
+            pageList.add(new PageObjects.EmbedPage(embedBuilder));
+        }
+        if (succeed.size() > 0) {
+            List<String> pageValues = PageUtils.splitString(succeed.stream().map(event -> Language.i18n(context.getLocale(), "enums.moldogevent." + event.name().toLowerCase() + ".display")).collect(Collectors.joining("\n")), 1000, '\n');
+            int subPage = 1;
+            for (String pageValue: pageValues) {
+                EmbedBuilder embedBuilder = new EmbedBuilder();
+                embedBuilder.setColor(MessageType.SUCCESS.getColor());
+                embedBuilder.setTitle("Disabled Events");
+                embedBuilder.setDescription("These events where successfully disabled in the specified channel");
+                embedBuilder.addField("Events", pageValue, false);
+                if (pageValues.size() > 1) {
+                    embedBuilder.addField("Sub Page", subPage + "/" + pageValues.size(), true);
+                    subPage++;
+                }
+                pageList.add(new PageObjects.EmbedPage(embedBuilder));
+            }
+        }
+        if (failedEvents.size() > 0) {
+            List<String> pageValues = PageUtils.splitString(failedEvents.stream().map(event -> Language.i18n(context.getLocale(), "enums.moldogevent." + event.name().toLowerCase() + ".display")).collect(Collectors.joining("\n")), 1000, '\n');
+            int subPage = 1;
+            for (String pageValue: pageValues) {
+                EmbedBuilder embedBuilder = new EmbedBuilder();
+                embedBuilder.setColor(MessageType.WARNING.getColor());
+                embedBuilder.setTitle("Failed to disable events");
+                embedBuilder.setDescription("These events failed to disable. This means they are probably not enabled in that channel");
+                embedBuilder.addField("Events", pageValue, false);
+                if (pageValues.size() > 1) {
+                    embedBuilder.addField("Sub Page", subPage + "/" + pageValues.size(), true);
```

Comment:
Subpage?

Reply:
Yep. it they enable/disable a lot of events it is entirely possible the different possible outcomes (so succeeded, failed, can't find) have more chars then can fit on a single page. So I added a sub page field. I should probably implement this into the main page system at some point.

Reply:
:+1: